app/controllers/foreman_bootdisk/disks_controller.rb

```diff
@@ -40,6 +40,23 @@ def full_host
       end
     end
 
+    def subnet
+      begin
+        subnet = @disk.try(:subnet)
```

Comment:
Sorry, wasn't really sure what you wanted me to say, I assumed they were things @GregSutcliffe or you had to fix.